HPCC-15285 Improve information when Compiling from a git repository

richardkchapman opened the pull request:
Add the name of the branch used into the workuning as an application option.

Signed-off-by: Richard Chapman rchapman@hpccsystems.com

ghalliday committed:
HPCC-15425 Improve the reported position for invalid string characters

Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

ghalliday committed:
HPCC-15425 Common up some code following review

Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

ghalliday committed:
HPCC-15443 Add an annotation onto a graph to indicate it is the loop body

Signed-off-by: Gavin Halliday <gavin.halliday@lexisnexis.com>

GordonSmith committed:
HPCC-15420 JS Graph Depth + No Subgraphs

Default depth to 999 when subgraphs are disabled.

Fixes HPCC-15420

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

GordonSmith committed:
HPCC-15444 Show Subgraph IDs

Increase click-able area for selection.

Fixes HPCC-15444

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

jakesmith committed:
HPCC-13763 Ensure non body subgraphs of loop are marked local

Also ensure graphs that were otherwise be executed without
global synchronization, are marked global if they create a result
and are a loop boty subgraph

Signed-off-by: Jake Smith <jake.smith@lexisnexis.com>

jamienoss committed:
HPCC-15446 Redis Plugin code typo

Signed-off-by: James Noss <james.noss@lexisnexis.com>

richardkchapman committed:
Merge pull request #8553 from jamienoss/issue15446-redis-plugin-pass-password-typo

HPCC-15446 Redis Plugin code typo

Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

jakesmith committed:
HPCC-15450 Ensure lookahead required count restartable

Signed-off-by: Jake Smith <jake.smith@lexisnexis.com>

richardkchapman committed:
HPCC-15285 Improve information when Compiling from a git repository

Add the name of the branch used into the workuning as an application option.

Signed-off-by: Richard Chapman <rchapman@hpccsystems.com>

richardkchapman commented:
@ghalliday Please review

HPCCSmoketest commented:
Automated Smoketest 
Sha: 4788bf1777d2eaa44860b69894d77f1116fd0780
Build: success
Install hpccsystems-platform-community_6.1.0-rc0.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

richardkchapman committed:
Merge pull request #8549 from GordonSmith/HPCC-15444

HPCC-15444 Show Subgraph IDs

Reviewed-By: Miguel Vazquez <miguel.vazquez@lexisnexis.com>
Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

richardkchapman committed:
Merge pull request #8548 from GordonSmith/HPCC-15420

HPCC-15420 JS Graph Depth + No Subgraphs

Reviewed-By: Miguel Vazquez <miguel.vazquez@lexisnexis.com>
Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

richardkchapman committed:
Merge pull request #8543 from ghalliday/issue15425

HPCC-15425 Improve the reported position for invalid string characters

Reviewed-By: Shamser Ahmed <shamser.ahmed@lexisnexis.co.uk>
Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

ghalliday committed:
Merge pull request #8554 from jakesmith/hpcc-13763

HPCC-13763 Ensure non body subgraphs of loop are marked local

Reviewed-by: Gavin Halliday <ghalliday@hpccsystems.com>

richardkchapman committed:
Merge pull request #8565 from ghalliday/issue15443

HPCC-15443 Add an annotation onto a graph to indicate it is the loop body

Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

g-pan committed:
HPCC-14680 DOCS:Add Security Manager doc

Fix HPCC-14680 DOCS:Add Security Manager doc
Add documentation for a pluggable security manager component.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-14680 DOCS:Add Security Manager doc

Fix HPCC-14680 DOCS:Add Security Manager doc
Add documentation for a pluggable security manager component.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

richardkchapman committed:
Merge pull request #8568 from g-pan/HPCC14680-SecMgr

HPCC-14680 DOCS:Add Security Manager doc

Reviewed-By: Jim DeFabia <james.defabia@lexisnexis.com>
Reviewed-By: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>
Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

richardkchapman committed:
Merge pull request #8555 from jakesmith/hpcc-15450

HPCC-15450 Ensure lookahead required count restartable

Reviewed-By: Richard Chapman <rchapman@hpccsystems.com>

jakesmith committed:
HPCC-15463 Avoid marking loop subgraphs global where unecessary

Ignore the information only dependency for the root loop body
subgraph, when marking up global result writer dependencies.

Signed-off-by: Jake Smith <jake.smith@lexisnexis.com>

ghalliday committed:
Merge pull request #8570 from jakesmith/hpcc-15463

HPCC-15463 Avoid marking loop subgraphs global where unecessary

Reviewed-by: Gavin Halliday <ghalliday@hpccsystems.com>

richardkchapman commented:
Closing to target 6.0.0
